Python-Function-Complex.md

```diff
@@ -0,0 +1,54 @@
+# Python Complex()
+
+`complex()` is a built-in function in Python 3 that hanldles the creation of a complex number according to the arguments passed. It returns a complex number with the value `real + imag*1j` when called in form of `complex(real, imag)` or converts a string or number to a complex number.
```

Comment:
Since this is a beginners' article, I would recommend adding a brief description of what a complex number is.

At the very least, where you first mention `complex`, link to some external wiki article (say Wikipedia) about what is a complex number.